oas-validator/oas-validator-compatibility/src/main/java/org/apache/servicecomb/toolkit/oasv/compatibility/factory/DefaultExampleDiffValidatorFactory.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.servicecomb.toolkit.oasv.compatibility.factory;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.apache.servicecomb.toolkit.oasv.diffvalidation.api.ExampleDiffValidator;
+import org.apache.servicecomb.toolkit.oasv.diffvalidation.factory.ExampleDiffValidatorFactory;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DefaultExampleDiffValidatorFactory implements ExampleDiffValidatorFactory {
```

Comment:
Why do we need a example diff validator?

Reply:
Actually we don't need it, I just create corresponding validators for every openapi object